incubator/elasticsearch/templates/data-statefulset.yaml

```diff
@@ -120,14 +120,23 @@ spec:
       - name: config
         configMap:
           name: {{ template "elasticsearch.fullname" . }}
+{{- if not .Values.data.persistence.enabled }}
```

Comment:
nit; this can be indented

Reply:
Sure thing. Will have an update for this.